File: PathoFroh/src/main/java/com/patho/main/util/dialogReturn/StainingPhaseUpdateEvent.java
package com.patho.main.util.dialogReturn;

import com.patho.main.model.patient.Task;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class StainingPhaseUpdateEvent {
	private Task task;
}

File: PathoFroh/src/main/kotlin/com/patho/main/util/event/dialog/StainingPhaseUpdateEvent.kt
package com.patho.main.util.event.dialog

import com.patho.main.model.patient.Task
import com.patho.main.util.dialogReturn.ReloadEvent

/**
 * Return event with a staining phase update
 */
class StainingPhaseUpdateEvent(val task: Task) : ReloadEvent()